src/prefect/cli/worker.py

```diff
@@ -181,6 +195,18 @@ async def start(
     app.console.print(f"Worker {worker.name!r} stopped!")
 
 
+async def _check_work_pool_paused(work_pool_name: str) -> bool:
+    try:
+        async with get_client() as client:
+            work_pool = await client.read_work_pool(work_pool_name=work_pool_name)
+            return work_pool.is_paused
+    except ObjectNotFound:
+        typer.secho(
+            f"Work pool {work_pool_name!r} does not exist.", fg=typer.colors.RED
+        )
```

Comment:
The worker will create a work pool if it doesn't exist, so we don't need to print this message.
```suggestion
```

Reply:
Okay make sense I will remove it .